cmd/ddev/cmd/composer-create.go

```diff
@@ -143,13 +145,16 @@ project root will be deleted when creating a project.`,
 			util.Failed("Failed to create project: %v", err)
```

Comment:
Isn't this message incorrect? Also, if it's just a fail to mv, probably not fatal any more?

Reply:
My thinking was that, at this point, the project hasn't been installed to the expected directory and likely isn't easily startable by a typical user, since the source code and docroot will be hidden inside a .tmp... dir. That would mean that the top-level intention of the command has failed, with the specifics detailed in the error text.

Reply:
Well, the mv will now be handled on the host, right, per other discussion. So we can give more details in the error reporting of that code.